pilot/pkg/model/service_test.go

```diff
@@ -192,11 +192,23 @@ func TestWorkloadInstanceEqual(t *testing.T) {
 		shouldEq bool
 		name     string
 	}{
+		{
+			comparer: nil,
+			comparee: nil,
+			shouldEq: true,
+			name:     "two null endpoints",
+		},
+		{
+			comparer: nil,
+			comparee: exampleInstance.DeepCopy(),
```

Comment:
I add the nil guard because `pc.c.workloadInstancesByPods[key]` would be nil when the pod becomes ready for the first time, and this is the util test to verify it works

Reply:
Oh I think it's not needed cos the ready event would be handled by endpoints controller, thanks for reminding, I will fix it later